electron/legendary/games.ts

```diff
@@ -389,15 +390,9 @@ class LegendaryGame extends Game {
     // not a perfect solution but it's the only choice for now
 
     // @adityaruplaha: this is kinda arbitary and I don't understand it.
-    //const pattern = process.platform === 'darwin' ? 'legendary' : this.appName
-
-    const { install : { install_path, executable} } = await this.getGameInfo()
-    const exe = install_path + '/' + executable
-    console.log('killing', this.appName)
-    return await execAsync(`pkill -ef ${exe}`).then((v) => {
-      this.state.status = 'done'
-      return v
-    })
+    const pattern = process.platform === 'darwin' ? 'legendary' : this.appName
+    console.log('killing', pattern)
+    return spawn('pkill', ['-f', pattern])
```

Comment:
This returns the ChildProcess, which doesn't match the type annotation.

Considering doing what I suggested in [#385 (comment)](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/385#issuecomment-857705880).

Reply:
Yeah but the issue is only the type?
Because I changed the type.
I think for stop, since we are killing the process, we don't need to use a Promise.
In the worst case, pkill might not exists on some distro and the command won't work, but we don't need to worry about every single use case and distros and configurations. We will go crazy like this.

Reply:
The type annotation doesn't seem to be fixed though.

Reply:
Ok, I fixed it to use spawn the right way, so the return of the function will be a console.log, a void.